### ISSUE TITLE
highlighted demo menu item loses its highlight when viewing a demo

### ISSUE THREAD

**Issue by agcolom** [jquery/jqueryui.com#58]
On jqueryui.com, click demos. The demos menu item is highlighted.
Now click on any element on the left hand-side menu e.g. Draggable, the demos menu item is no longer highlighted, so the user has no way of knowing he actually is under the demos menu. 

This does not happen on the api docs. When that menu item is selected, it gets highlighted and remains so when an item in the left hand-side menu is selected.

As long as the user is under the demos menu item, that menu item should be highlighted

**Comment by ashg1910** [comment]
@agcolom This goes with development, support and about as well. I can try to fix this

**Comment by scottgonzalez** [comment]
@ashg1910 I'm not sure what you're referring to, none of those have sub-pages.

**Comment by ashg1910** [comment]
@scottgonzalez When you click on, say 'Support', it gets highlighted and then if you click on any item on the left menu, the highlight is removed.

**Comment by scottgonzalez** [comment]
That's correct. You're no longer in the support section, which is a single page.

**Comment by ashg1910** [comment]
So this needs to be corrected for demos subsection only?

**Comment by scottgonzalez** [comment]
Yes, but it's important to note that there's no actual demo section. There is a single page which just lists all of the modules. This is done purely to provide content at a reasonably expected URL of jqueryui.com/demos/.
